pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiValidation.java

```diff
@@ -10,6 +10,7 @@
 import org.jdbi.v3.sqlobject.statement.SqlBatch;
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
 
+
```

Comment:
Totally bogus change so there is a Java code change that will allow the execution of the tests.